build.py

```diff
@@ -97,10 +106,30 @@ def write(self, outputdir):
         self._write_v2(outputdir / "v2")
 
     def _write_v1(self, outputdir):
+        self._validate_config_v1_aginst_schema()
         logger.info("Writing v1 configs")
         outputdir.mkdir(parents=True, exist_ok=True)
         self._write_configs(outputdir, self.configs_v1)
 
+    def _validate_config_v1_aginst_schema(self):
+        registry = Registry(retrieve=retrieve_schema)
+        # Logging the base URI for the schema directory
+        logger.info("Validation of generated rules.json against schema")
```

Comment:
This message is a statement, and an incomplete one, IMHO. In a statement like this, I would expect to see indication of the result - did the validation pass or fail?

All other info messages that inform about what the tool will do next use a very specific form:

* They use a verb in present participle to indicate that the action described by the message has not finished yet. 
* They have the form "<action>: <object of the action>" so that both the action description and the object are easy to identify. Messages that contain long paths in the middle are difficult to read.

I don't think that we need to log success explicitly.  We do log errors. No error implicitly means that the action succeeded.

Having said that, I would prefer the message to look something like this

```suggestion
        logger.info(f"Validating generated file against {schema_name}: {config_path}")
```

Note that other info messages use absolute paths so this one should, too.

Reply:
Fixed